[#10779] Docs for PITR delete_snapshot_schedule (#11900)
yugabyte/yugabyte-db: docs/content/latest/admin/yb-admin.md

```output
{
    "snapshot_id": "6eaaa4fb-397f-41e2-a8fe-a93e0c9f5256",
    "restoration_id": "b1b96d53-f9f9-46c5-b81c-6937301c8eff"
}
```

#### delete_snapshot_schedule

> Deletes the snapshot schedule with this ID. Note, this will also remove all of the snapshots associated with the schedule!

Reviewer: ```suggestion
Deletes the snapshot schedule with the given ID, **and all of the snapshots** associated with that schedule.
```

Returns a JSON object with the schedule_id that was just deleted.

**Syntax**

```sh
yb-admin delete_snapshot_schedule <schedule-id>
```

Where _schedule-id_ is the snapshot schedule's unique identifier.

**Example**

```sh
$ ./bin/yb-admin \
    delete_snapshot_schedule 6eaaa4fb-397f-41e2-a8fe-a93e0c9f5256
```

The output should show the snapshot ID we just deleted.

> ```output

Reviewer: Fix the markdown, and give the JSON some highlighting...
````suggestion

```output.json
````
